astropy/io/fits/tests/test_connect.py

```diff
@@ -897,31 +891,6 @@ def test_fits_mixins_per_column(table_cls, name_col, tmpdir):
         assert t2[name]._time.jd2.__class__ is np.ndarray
 
 
-def test_warn_for_dropped_info_attributes(tmpdir, monkeypatch):
-    # make sure that yaml cannot be imported if it is available
-    monkeypatch.setitem(sys.modules, 'yaml', None)
-
-    filename = str(tmpdir.join('test.fits'))
-    t = Table([[1, 2]])
-    t['col0'].info.description = 'hello'
-    with pytest.warns(AstropyUserWarning, match=r"table contains column\(s\) "
-                      "with defined 'format'") as warns:
-        t.write(filename, overwrite=True)
-    assert len(warns) == 1
-
-
-def test_error_for_mixins_but_no_yaml(tmpdir, monkeypatch):
-    # make sure that yaml cannot be imported if it is available
-    monkeypatch.setitem(sys.modules, 'yaml', None)
```

Comment:
lint: Since this change, the `sys` module is actually no longer used in this module, so its import can be removed.

Reply:
Done!